conformance/eexp/arg_inlining.ion

```diff
@@ -0,0 +1,15 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Inlining of multiple return values into E-expression arguments.
+
+
+(ion_1_1
+  (each (toplevel ('#$:$ion:make_list' 1 2 ('#$:values') 3 4))
```

Comment:
Any particular reason that `make_list` has a qualified name, but `values` does not? Should we have different test cases for checking that the qualified and unqualified names are equivalent?

Reply:
No particular reason.  These were early test cases so I was trying lots of stuff at once. 

I agree that they should be consistent and tests for name qualification should be separate.

Reply:
Pushed a change removing the qualification and adding a description to this test.